scripts/webpack-build-stats

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+export NODE_ENV=production
```

Comment:
yes `export` sets the variable globally, we can just remove the `export` to make it local. If you want to get nuts you can do `local NODE_ENV=production`.